assets/js/components/SaptuneDetails/SaptuneServiceStatus.test.jsx

```diff
@@ -0,0 +1,117 @@
+import React from 'react';
+import { screen } from '@testing-library/react';
+import '@testing-library/jest-dom';
+
+import { renderWithRouter } from '@lib/test-utils';
+
+import SaptuneServiceStatus from './SaptuneServiceStatus';
+
+describe('SaptuneServiceStatus', () => {
+  it.each([
+    {
+      service: { name: 'saptune', enabled: 'enabled', active: 'active' },
```

Comment:
I don't think we need this `service` subtree.
Just `serviceName`, `enabled` and `active` should be alright.
Once you adjust the code to get the `serviceName` it will be ok